src/main/webapp/assets/css/global-custom.css

```diff
@@ -488,3 +488,38 @@ body{
 .faq .accordion .question .question-heading {
     padding-right: 30px;
 }
+
+/*upcoming events card*/
+.card-event{
+    border-radius: 1rem;
+}
+
+.card-event .card-body .card-event-date{
+    border: 0.0625rem solid #fbc53b;
+    border-radius: 50%;
+    padding: .5rem .75rem;
+}
+
+.after-content{
+    display: none;
+}
+
+.hover-trigger:hover .after-content {
+    display: block;
+}
+
+.hover-trigger:hover .original-content {
+    display: none;
+}
+
```

Comment:
Upcoming events?

Reply:
Also, let's add some meaningful CSS selectors.

Reply:
What do you suggest? 
Just so that I can learn, What are the mistakes that I have done in naming these selectors?

Reply:
1. Rename the comment with a relevant one (It's not an upcoming event)
2. Let's replace `.card-event` with `.onelive-video-card` or just `.video-card`
3. Wrap every other selector with `.card-event`
4. `.after-content` is little bit unclear

Reply:
Thank you @jayasanka-sack, You are the best.